lalrpop/src/lib.rs

```diff
@@ -6,6 +6,9 @@
 // too darn annoying to try and make them do so.
 #![cfg_attr(test, allow(dead_code))]
 
+#![feature(in_band_lifetimes)]
+#![feature(infer_struct_outlives)]
```

Comment:
Were you testing something here? (Hope this isn't required for anything).

Reply:
Oh yeah, let me revert that :)